src/devtools/devtoolsMiddleware.ts

```diff
@@ -1,24 +1,71 @@
-let devTools = { instance: null };
+import Store from "../interfaces/Store";
```

Comment:
What about moving all those Interfaces to a different file?

Maybe create an interface/ folder inside devtools/ folder, one interface = one new file.

What do you think?

Reply:
done!